data/core/units/humans/Outlaw_Shadow_Lord.cfg

```diff
@@ -0,0 +1,148 @@
+#textdomain wesnoth-units
+
+#define ALTERNATIVE_DAMAGE_TYPE PRIORITY TYPE
+    [damage_type]
+        [filter_opponent]
+            formula=" resistance.{TYPE}=min(values(resistance))" # and not min(values(resistance))>0
+        [/filter_opponent]
+        replacement_type={TYPE}
+        overwrite_specials=one_side
+        [overwrite]
+            priority={PRIORITY}
+        [/overwrite]
+    [/damage_type]
+#enddef
+
+[unit_type]
+    id=Shadow Lord
+    name= _ "Shadow Lord"
+    race=human
+    gender=male,female
+    image="units/human-outlaws/shadow-lord.png"
+    hitpoints=41 # great offensive power, but less hp than a lvl 1 troll whelp
+    [abilities]
+        {ABILITY_NIGHTSTALK}
+    [/abilities]
+    movement_type=smallfoot
+    movement=6
+    experience=150
+    level=3
+    alignment=liminal
+    advances_to=null
+    cost=71
+    {AMLA_DEFAULT}
+    usage=mixed fighter
+    description= _ "The fruits of the paths of either light or darkness are well known to magi: the brilliance of a white mage or the deep gloom of a necromancer. Much less well known is the art of the Shadow Lord, those few who master a union of both white and black magic. Though decades of zealous study and self-neglect have left them drained of strength, these savants have transcended the need for crude physical weapons, instead channeling their energies into devastating magical attacks that undercut their enemies’ resistances."
+    die_sound={SOUND_LIST:HUMAN_DIE}
+    [standing_anim]
+        start_time=0
+        [frame]
+            image="units/human-outlaws/shadow-lord.png:70,units/human-outlaws/shadow-lord-[1~7].png:70"
+        [/frame]
+    [/standing_anim]
+    {DEFENSE_ANIM "units/human-outlaws/shadow-lord-defend2.png" "units/human-outlaws/shadow-lord-defend1.png" {SOUND_LIST:HUMAN_OLD_HIT} }
+    [attack]
+        name=astral blade
+        icon="icons/sword-astral.png"
```

Comment:
This attack icon wasn't transferred over.

Reply:
Fixed, thanks